core/ecschema-metadata/test/UnitConversion/MetricUnits.ecschema.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Put the schemas in an 'assets' directory ... see examples in other packages test directories.  Doing this keeps the tests and test input data separate